cloudbuild-cache.yaml

```diff
@@ -10,17 +10,17 @@ steps:
     path: /root/.ssh
   id: 'untar-mod-cache'
 
-- name: 'golang:1.16.3'
-  args: ['go', 'mod', 'download']
+- name: 'golang:1.18.1'
```

Comment:
do we want to do 1.18.2 to match what we use in other repos?

Reply:
good call! updated